addons/stock/tests/test_move.py

```diff
@@ -2000,6 +2000,20 @@ def test_availability_8(self):
         move_partial._action_assign()
         self.assertEqual(move_partial.state, 'assigned')
 
+    def test_product_tree_views(self):
```

Comment:
Is this test failing without the groups?

Reply:
If so, please put a docstring about the intention of the test.

Reply:
Exactly, this test would fail without `groups`. I created it because I think it is a way to check that there is no error.